topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorMockTest.java

```diff
@@ -90,7 +88,10 @@ public static void after() throws InterruptedException {
     }
 
     @BeforeEach
-    public void createMockKube(VertxTestContext context) throws Exception {
+    public void setup(VertxTestContext context) throws Exception {
```

Comment:
Create cluster in `@BeforeEach` instead of `@BeforeAll` once checkpoints fixed up, tests were failing due to `my-topic` already existing.

Reply:
:+1: